vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -113,25 +113,25 @@ func NewVolumeDriver(targetURL string, projectID string, hostID string, mountDir
 // map hasn't been initialized, return 1 prevent detach and remove.
 
 // Return the number of references for the given volume
-func (d *VolumeDriver) getRefCount(vol string) uint {
+func (d *VolumeDriver) getRefCount(vol string) (uint, error) {
```

Comment:
I would keep the current code AS-IS. Ref count not initialized is not really an error. You can get away with specific error message by modifying "Remove". See comment below.